pipeline-steps/pubcloud.groovy

```diff
@@ -121,7 +122,11 @@ def delPubCloudSlave(Map args){
     step_name: 'Cleanup',
     step: {
       ssh_slave.destroy()
-      cleanup()
+      cleanup (
```

Comment:
The cleanup function needs to be cleanup(Map args){ in order to accept these args. Also other jobs (eg RPC-AIO) that call cleanup will need to be modified to pass the right args.